python/ray/util/scheduling_strategies.py

```diff
@@ -2,26 +2,17 @@
 from ray.util.annotations import PublicAPI
 from ray.util.placement_group import PlacementGroup
 
+# The default hybrid scheduling strategy
+# based on config scheduler_spread_threshold.
+# This disables any potential placement group capture.
+DEFAULT_SCHEDULING_STRATEGY = "DEFAULT"
 
-@PublicAPI(stability="beta")
-class SchedulingStrategy(object):
-    """Base class for all scheduling strategies.
-    """
-    pass
-
-
-@PublicAPI(stability="beta")
-class DefaultSchedulingStrategy(SchedulingStrategy):
-    """The default hybrid scheduling strategy
-       based on config scheduler_spread_threshold.
-
-    This disables any potential placement group capture.
-    """
-    pass
+# Spread scheduling on a best effort basis.
+SPREAD_SCHEDULING_STRATEGY = "SPREAD"
 
 
 @PublicAPI(stability="beta")
-class PlacementGroupSchedulingStrategy(SchedulingStrategy):
+class PlacementGroupSchedulingStrategy(object):
```

Comment:
FYI @jjyao, you don't need to subclass `object` in Python 3 to get a new-style class, and now that Python 2 has reached EOL, it's considered best practices to not subclass `object`: `class PlacementGroupSchedulingStrategy:`

Reply:
ok, I'll change it in my next PR.